ghwatch.py

```diff
@@ -33,6 +33,9 @@
     # Repository with github metadata mirror (to get label data)
     'meta': {'bitcoin/bitcoin': '/path/to/bitcoin-gh-meta'},
 
+    # Interval in seconds for an auto-update (git pull) of github metadata mirror if greater than 0.
+    'auto-update': 0,
```

Comment:
Might want to make this `auto_update`—no strong opinion on snake case versus kebab case but it's good to be consistent with `label_prio`.